api/models/event.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+import("time")
+
+// TODO: decide which fields are nullable.
+type Event struct {
+	EventID			int			`json:"eventID"`
```

Comment:
I just realized that calling `Event.EventName` might be a bit redundant in both Go code and SQL queries. Do you think it's fine or should we change the schema to be:
```Go
type Event struct {
  ID
  Category
  Name
  Location
  Date
}
```
?

Reply:
The problem is some of that are keywords in SQL and I believe that in itself is bad practice

Reply:
Fair. what about keeping the schema in postgres but changing the model fields in Go? 
If you think it would get confusing I don't mind leaving as it is. A bit of redundancy is not a big deal

Reply:
I can see both arguments as having the names be different may between the DB and model can get confusing as the complexity increases so the redundancy is fine with that point. But, it does make it redundant while writing the code.

Reply:
Let's keep it like this for now then